main.go

```diff
@@ -64,32 +68,49 @@ func main() {
 			log.Fatalf("must supply --tls-certificate with --tls-private-key")
 		}
 	}
-
-	srv, err := server.NewFromListener(listener)
-	if err != nil {
-		log.Fatalf("failed to create server: %v", err)
-	}
+	var grpcServer = grpc.NewServer(
+		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
```

Comment:
Are we collecting prometheus metrics anywhere?

Reply:
I don't think they're currently exposed. I just created #15 for that. Thanks for pointing this out.